src/dsp_tools/xmllib/models/root.py

```diff
@@ -214,13 +204,17 @@ def write_file(self, filepath: str | Path, default_permissions: Permissions | No
             ```python
             root.write_file("xml_file_name.xml")
             ```
-
-            ```python
-            # To overwrite `Permissions.PROJECT_SPECIFIC_PERMISSIONS`
-            root.write_file("xml_file_name.xml", Permissions.PUBLIC)
-            ```
         """
-        root = self.serialise(default_permissions)
+
+        if default_permissions:
+            msg = (
+                "You added a default permission, this functionality is deprecated "
+                "as project wide default permissions are set in the json. "
+                "This parameter will be removed soon."
```

Comment:
```suggestion
                "You added a default permission. This has no effect. This functionality is deprecated, "
                "because project wide default permissions are set in the JSON project file. "
                "This parameter will be removed soon."
```